examples/notebooks/beam-ml/bigtable_enrichment_transform.ipynb

```diff
@@ -0,0 +1,854 @@
+        "                                          model_class=build_model,\n",
+        "                                          model_params={'n_inputs':5, 'n_outputs':1}\n",
+        "                                          ).with_preprocess_fn(convert_row_to_tensor)"
+      ],
+      "metadata": {
+        "id": "VqUUEwcU-r2e"
+      },
+      "execution_count": 20,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Define a `DoFn` to format the output."
+      ],
+      "metadata": {
+        "id": "vNHI4gVgNec2"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "class PostProcessor(beam.DoFn):\n",
+        "  def process(self, element, *args, **kwargs):\n",
+        "    print('Customer %d who bought product %d is recommended to buy product %d' % (element.example[3], element.example[0], math.ceil(element.inference[0])))"
+      ],
+      "metadata": {
+        "id": "rkN-_Yf4Nlwy"
+      },
+      "execution_count": 21,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0a1zerXycQ0z"
+      },
+      "source": [
+        "## Run the pipeline\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Configure the pipeline for streaming mode."
+      ],
+      "metadata": {
+        "id": "WrwY0_gV_IDK"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "options = pipeline_options.PipelineOptions()\n",
+        "options.view_as(pipeline_options.StandardOptions).streaming = True # Streaming mode is set True"
+      ],
+      "metadata": {
+        "id": "t0425sYBsYtB"
+      },
+      "execution_count": 22,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Pub/Sub sends the data in bytes. Convert the data to `beam.Row` objects by using a `DoFn`."
+      ],
+      "metadata": {
+        "id": "DBNijQDY_dRe"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "class DecodeBytes(beam.DoFn):\n",
+        "  \"\"\"\n",
+        "  The DecodeBytes `DoFn` converts the data read from Pub/Sub to `beam.Row`.\n",
+        "  First, decode the encoded string. Convert the output to\n",
+        "  a `dict` with `json.loads()`, which is used to create a `beam.Row`.\n",
+        "  \"\"\"\n",
+        "  def process(self, element, *args, **kwargs):\n",
+        "    element_dict = json.loads(element.decode('utf-8'))\n",
+        "    yield beam.Row(**element_dict)"
+      ],
+      "metadata": {
+        "id": "sRw9iL8pKP5O"
+      },
+      "execution_count": 23,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Use the following code to run the pipeline.\n",
+        "\n",
+        "**Note:** Because this pipeline is a streaming pipeline, you need to manually stop the cell. Otherwise, the pipeline continues to run."
```

Comment:
```suggestion
        "**Note:** Because this pipeline is a streaming pipeline, you need to manually stop the cell. If you don't stop the cell, the pipeline continues to run."
```